### PR TITLE
Instructions panel Update

### PR DESCRIPTION
**Pull request by Joeper214**
Please Review and Merge PR. @root-beer thanks! :bow:

### CODE REVIEW

**Comment by root-beer** [comment]
I'll review now.

File: config/locales/en.yml [hunk 1]
```diff
@@ -714,5 +714,5 @@ en:
     cf_template_head: 'Apply CloudFormation Template'
     cf_template_body: |
       In order to apply a cloud formation template, you have to <b> select an infrastructure </b> to show details and click the <b> operations menu</b> and apply <b> cloudformation tab </b>.
-      Then, the list of cloud formation templates will appear in the screen. Please select the template as you like and fillout the required fileds to set the parameters then press end to start creating your new stack.
+      Then, the list of cloud formation templates will appear in the screen. Please select the template as you like and fillout the required fields to set the parameters then press send to start creating your new stack.
```
Comment (Joeper214) [inline config/locales/en.yml]: 🎯

**Comment by root-beer** [comment]
LGTM! Thanks. 🙇

**Comment by Joeper214** [comment]
:bow:

**Pull request merged**